gaffer-core/common-util/src/main/java/gaffer/commonutil/StreamUtil.java

```diff
@@ -37,6 +37,9 @@
     public static final String STORE_TYPES = SCHEMA_FOLDER + "storeTypes.json";
     public static final String STORE_PROPERTIES = "/store.properties";
     public static final String OP_AUTHS = "/opAuths.properties";
+    public static final String OP_SCORES = "/opScores.properties";
+    public static final String ROLE_SCORES = "/roleScores.properties";
```

Comment:
I think that makes it more confusing, its a file of authorisation scores. A max of these is taken but for the actual properties file thats irrelevant its just an auth has a score.